hack/install-etcd.sh

```diff
@@ -18,11 +18,11 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-KUBE_ROOT=$(dirname "${BASH_SOURCE}")/../..
+KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 
-ETCD_VERSION=${ETCD_VERSION:-v2.0.0}
+ETCD_VERSION=${ETCD_VERSION:-v2.0.12}
```

Comment:
Another instance of this literal.  I know it's going to be a bit messy to implement, but can we not specify it canonically somewhere?